internal/helmrelease/helmrelease.go

```diff
@@ -0,0 +1,100 @@
+package helmrelease
```

Comment:
This package can be deleted from your branch now that your code is referencing the stuff in `pkg/apis`